games-util/lutris/lutris-0.4.23.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit distutils-r1 gnome2-utils python-r1
```

Comment:
I don't see a reason you need to inherit both distutils and python-r1, if it works with distutils.